modules/images-configuration-create-config-map.adoc

```diff
@@ -0,0 +1,61 @@
+[id="steps-for-manual-configmap-creation"]
+= Steps to create the image signature ConfigMap
+
+The following commands will create and apply the image signature ConfigMap to the cluster you want to update. 
+
+.Procedure
+
+. Find the link:https://access.redhat.com/solutions/4583231[version] you want the cluster to update.
+
+. Add the version to OCP_RELEASE_NUMBER environment variable.
++
+----
+$ OCP_RELEASE_NUMBER=<{product-version}.0>
+----
```

Comment:
Need step to set ARCHITECTURE and a ref back to the solution article, or something up front that keys them into what they are taking away from the solution article, so they know how to set it.

Reply:
Added the architecture and a note, but did not added much info around what they need from the KB article as this is a module and will be called from the update doc which would have the context.  Also the KB article has some information.